### PR TITLE
Up and Down voting

### PR DESCRIPTION
**Pull request by mkapolka**
emilllyyyyyyyyyyyy

### CODE REVIEW

File: sutrofm/redis_models.py [hunk 1]
```diff
@@ -206,7 +212,7 @@ def get(connection, party_id, id):
       # Filter out empty strings
       output.upvotes = set(filter(None, output.upvotes))
       output.downvotes = set(filter(None, output.downvotes))
-      output.timestamp = parser.parse(data.get('timestmap', datetime.datetime.utcnow().isoformat()))
```
Comment (superemily) [inline sutrofm/redis_models.py:209]: NICE WORK!

File: sutrofm/static/js/search-view.js [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
     enqueueTrack: function(trackKey) {
       $.ajax({
-        'url': '/api/party/'+window.roomId+'/enqueue',
+        'url': '/api/party/'+window.roomId+'/queue/enqueue',
```
Comment (superemily) [inline sutrofm/static/js/search-view.js:17]: this should be add but i'm totally accepting this

File: sutrofm/urls.py [hunk 3]
```diff
@@ -22,9 +22,11 @@
   url(r'^create-auth/', 'sutrofm.views.createauthtoken'),
 
   url(r'^api/parties/$', api_views.parties, name='api_parties'),
-  url(r'^api/parties/((?P<party_id>[A-Za-z0-9\-_]+)/)?$', api_views.get_party_by_id),
-  url(r'^api/party/((?P<party_id>[A-Za-z0-9\-_]+)/)queue$', api_views.get_party_queue),
-  url(r'^api/party/((?P<party_id>[A-Za-z0-9\-_]+)/)enqueue$', api_views.add_to_queue),
+  url(r'^api/party/(?P<party_id>[A-Za-z0-9\-_]+)/?$', api_views.get_party_by_id),
```
Comment (superemily) [inline sutrofm/urls.py:25]: These are really hard to read.

File: sutrofm/static/js/search-view.js (hunk 2 above)
Comment (mkapolka) [inline sutrofm/static/js/search-view.js:17]: actually this is api spec https://github.com/sutrofm/sutrofm/wiki/Backend-Re-Write

File: sutrofm/urls.py (hunk 3 above)
Comment (mkapolka) [inline sutrofm/urls.py:25]: yeah if you're a n00b

**Pull request merged**